addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -725,14 +725,18 @@ blockquote {
     }
 }
 %o-we-background-layer {
-    @include o-position-absolute(0, 0, 0, 0);
+    position: absolute !important;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
     display: block;
     overflow: hidden;
     background-repeat: no-repeat;
     pointer-events: none;
 }
 
-section {
+section, .oe_img_bg, [data-oe-shape-data], .o_background_video, .parallax {
```

Comment:
- background video is not a class of we_editor
- I made a localized fix for the Big Boxes snippet, I guess your fix is better though so you can remove my fix at the same time in this commit